tweet_bot.py

```diff
@@ -4,24 +4,51 @@
 from credentials import *
 import os.path
 
-auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
-auth.set_access_token(access_token, access_token_secret)
-api = tweepy.API(auth)
-time_delay = 12*60*60 #12 hours delay
+def get_def():
+	file = open('dir_to_lookup/barrons800.txt', 'r')
+	data = file.readlines()
+	while True:
+		to_send = random.choice(data).strip()
+		if len(to_send) < 280 and len(to_send) > 5:
+			break
+	return to_send
 
-#api.update_status('testing a twitter bot. continue scrolling.')
+def main():
+	auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+	auth.set_access_token(access_token, access_token_secret)
+	api = tweepy.API(auth)
 
-def get_def():
-    file = open('dir_to_lookup/barrons800.txt', 'r')
-    data = file.readlines()
-    while True:
-        to_send = random.choice(data).strip()
-        if len(to_send) < 280 and len(to_send) > 5:
-            break
-    return to_send
-
-while True:
-    to_write = get_def()
-    print(to_write)
-    api.update_status(to_write)
-    time.sleep(time_delay)
\ No newline at end of file
+	no_of_words_per_day = 6
+	#time_delay = (12*60*60) // no_of_words_per_day
+	time_delay = 2
+	word_count = 0;
+	todays_words = []
+
+	while True:
+		if word_count < no_of_words_per_day:
+			to_write = get_def()
+			print(to_write)
+			#tweet = api.update_status(to_write)
+			#tweet_id = tweet.id_str
+			tweet_id = 1
+			word = (tweet_id, to_write.split(':')[0].strip())
+			todays_words.append(word)
+			word_count += 1
+			time.sleep(time_delay)
+
+		elif word_count == no_of_words_per_day:
+			print('\ntodays_words:',todays_words)
+			question = list(random.choice(todays_words))
+			print('random word:', question[1])
+
+			with open("answers.txt", "a") as file:
```

Comment:
Could you explain how the answer system works here?

Reply:
that's great . The code to check the comments on the tweet needs to be written now